docs/readme.md

```diff
@@ -12,7 +12,7 @@ const lighthouse = require('lighthouse');
 const chromeLauncher = require('chrome-launcher');
 
 function launchChromeAndRunLighthouse(url, flags = {}, config = null) {
-  return chromeLauncher.launch().then(chrome => {
+  return chromeLauncher.launch(flags).then(chrome => {
```

Comment:
launch() takes an opts object

a prop of that object is `chromeFlags` .. so perhaps rename `flags` to `launchOpts` and add in a `chromeFlags: ['--disable-popup-blocking']` into it for a clear example?